console/jsre/deps/doc.go

```diff
@@ -18,5 +18,18 @@
 // This file is derived from internal/jsre/deps/deps.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package deps contains the console JavaScript dependencies Go embedded.
+/*
+Package deps contains the console's dependencies of Javascript libraries which is to be embedded in Go source code.
+
+Klaytn console provides functions of bignumber.js and web3.js which are included in the source tree. To make those available, deps is using go-bindata<https://github.com/jteeuwen/go-bindata> to make those javascript source files into assets which can be used by go environment.
```

Comment:
```suggestion
Klaytn console provides functions of bignumber.js and web3.js which are included in the source tree. deps is using go-bindata<https://github.com/jteeuwen/go-bindata> to make those javascript source files into assets which can be used by go environment.
```
I think the changed part is redundant.